__tests__/integration/utils/wallet-helper.js

```diff
@@ -82,6 +153,8 @@ export class WalletHelper {
     };
   }
 
+  __setStarted() { this.#started = true; }
```

Comment:
Out of curiosity, why two `_`s?

Reply:
This is [another convention](https://www.thoughtco.com/and-in-javascript-2037515) for _private_ or _hidden internal methods_ on javascript.

Since I implemented the `startMultipleWalletsForTest`, I had to start a wallet from an outside context, but I didn't want this method to show up on code completion for developers. So, it's a kind of a _soft encapsulation_.

If it seems too weird for the current code style I can revert it to just `setStarted()`